lib/filter/filter-expr-grammar.ym

```diff
@@ -144,9 +140,24 @@ filter_simple_expr
 	| filter_re					{ $$ = last_filter_expr; }
 	| filter_plugin
 	| filter_comparison
-	| filter_throttle   { $$ = last_filter_expr; }
+	| filter_rate_limit_compat
 	;
 
+filter_rate_limit_compat
```

Comment:
this rule could be merged into the filter_plugin to avoid code duplication, e.g. create a non-terminal that is either LL_IDENTIFIER or KW_THROTTLE and use that in the filter_plugin rule.

Reply:
When KW_THROTTLE is found in this context, the name of the plugin we are looking for has to be changed.
But I think I can do that without code duplication, let me try it.

Reply:
Fixed.